Readme.md

```diff
@@ -203,8 +203,8 @@ that you can start or bind or map or whatever you want to do. To handle errors,
 for instance, we could do the following:
 
 ```swift
-provider = ReactiveSwiftMoyaProvider<GitHub>()
-provider.request(.userProfile("ashfurrow")).start { event in
+provider = MoyaProvider<GitHub>()
```

Comment:
Oh, this is probably more of a preference and I don't mind both - just the general assumption and code style is to rather preserve the original object and use extension when needed. Also, I wouldn't worry so much about forgetting to use `rx` or `reactive`, since these are really extensively used by people who use reactive programming with ReactiveSwift or RxSwift.